enhancements/security/runtime-class-scc.md

```diff
@@ -0,0 +1,290 @@
+# openshift scc runtime class enhancement
+---
+title: gate-runtime-classes-with-scc
+authors:
+  - "@haircommander"
+  - "@mrunalp"
+reviewers:
+  - "@deads2k"
+  - "@sttts"
+approvers:
+  - ""
```

Comment:
would like to see myself and tech lead of node team here.

Reply:
added you, @deads2k and @mrunalp